### PR TITLE
Add nupkg generation to CI

### PR DESCRIPTION
**Pull request by TOTBWF**
This PR adds a nupkg to the appveyor artifacts, fixing #170.

### CODE REVIEW

**Comment by TOTBWF** [comment]
I have no idea why this is failing, any thoughts?

**Comment by wallymathieu** [comment]
Could be that the ordering has changed due to changes in how FAKE orders dependencies. Since it tries to reorder based on the information provided. My guess is that the old way of having a long chain of dependencies that assume a certain order is faulty. FAKE might want to run things in parallel.

**Comment by wallymathieu** [comment]
Note the
```
Shortened DependencyGraph for Target BuildPackage:
```

**Comment by wallymathieu** [comment]
Looks like issue with doc generation https://github.com/fsprojects/Paket/issues/1295, I'll need to reproduce it.

**Comment by TOTBWF** [comment]
Seeing as we already build the docs in travis, it might make sense to just skip the step entirely on appveyor for now. As an aside, we should also probably make sure to bump the package version when doing this

File: build.fsx [hunk 1]
```diff
@@ -433,10 +433,10 @@ open Fake.Core.TargetOperators
   ==> "Build"
   ==> "CopyBinaries"
   ==> "RunTests"
-  ==> "GenerateReferenceDocs"
-  ==> "GenerateDocs"
   ==> "NuGet"
   ==> "CopyNuGet"
+  ==> "GenerateReferenceDocs"
+  ==> "GenerateDocs"
```
Comment (wallymathieu) [inline build.fsx]: Here you say that BuildPackage depends on GenerateDocs. Perhaps better to move GenerateReferenceDocs and GenerateDocs to a separate chain and let it only depend on CopyBinaries, then add a dependency on GenerateDocs from All.

Comment (TOTBWF) [reply build.fsx]: That seems more reasonable, will do

**Comment by wallymathieu** [comment]
The failure on travis looks unrelated, I'll create issue around the doc generation. Thanks for the contribution @TOTBWF 😄

**Pull request merged**